src/build_packs/nodejs.js

```diff
@@ -207,4 +211,114 @@ module.exports = class NodejsBuildPack extends BuildPack {
   getLibInstallCmd(opts) {
     return `npm install --save ${opts.libPkgName}`;
   }
+
+  bump(opts) {
+    if (this.config.global.dryRun) {
+      utils.logger.log('bump', 'Beginning dry run.'.cyan);
+    }
+
+    let semverLevel = opts['_'][1];
+    if (semverLevel === undefined) {
+      throw new Error('Parameter is required: major|minor|patch');
+    }
+    if (!['major', 'minor', 'patch'].includes(semverLevel)) {
+      throw new Error(
+        `Bad parameter ${semverLevel}, must be one of: major|minor|patch`
+      );
+    }
+
+    this.expandConfig(opts);
+
+    let oldVersion = this.config.pkgjson.version;
+    let name = this.config.pkgjson.name;
+    let newVersion = semver.inc(oldVersion, semverLevel);
+
+    utils.logger.log(
+      'bump',
+      `Version will be bumped from ${oldVersion.yellow} to ${
+        newVersion.yellow
+      }.`
+    );
+
+    let samplesPackageJson;
+    let samplesPackageJsonPath = path.join(
+      this.config.global.localPath,
+      'samples',
+      'package.json'
+    );
+    if (fs.existsSync(samplesPackageJsonPath)) {
+      try {
+        samplesPackageJson = JSON.parse(
+          fs.readFileSync(samplesPackageJsonPath)
+        );
+      } catch (err) {
+        throw new Error(
+          `Cannot parse samples package.json located at ${samplesPackageJsonPath}: ${err.toString()}`
+        );
+      }
+    }
+
+    utils.logger.log(
+      'bump',
+      `Version in ${'package.json'.yellow} will be set to ${newVersion.yellow}.`
+    );
+    if (samplesPackageJson !== undefined) {
+      let dependency = `'${name}': '${newVersion}'`;
+      utils.logger.log(
+        'bump',
+        `${'samples/package.json'.yellow} will depend on ${dependency.yellow}.`
+      );
+    }
+
+    if (opts.dryRun) {
+      utils.logger.log('bump', 'Dry run complete.'.cyan);
+      return;
+    }
+
+    let newPackageJson = extend(true, {}, this.config.pkgjson);
```

Comment:
I believe there's a method in lodash you can use to do this, and you can eliminate the dependency on `extend`.

Reply:
I probably can just edit `this.config.pkgjson` without copying it to the new object - originally I was not sure if it's going to break anything, but I now think it should be perfectly OK. What do you think?

Reply:
sgtm